### ISSUE TITLE
ACC-255.5 (Level AA), ACC-264.1 (Level A) _ Modify Modules _ Image of text _ Text alt not descriptive and text within image not read 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3135]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Image of text
Affected Check Point:	ACC-255.5, ACC-264.1
Short text:	ACC-255.5 (Level AA), ACC-264.1 (Level A) _ Modify Modules _ Image of text _ Text alt not descriptive and text within image not read 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Now navigate to ‘SAP Kyma’ logo within header shell bar region and mouse hover over it to see the text alternative.

3. Issue Type: Manual
Observed Behavior:
The text alternative ‘Logo’ provided for ‘SAP Kyma’ logo within header shell bar region  is not descriptive as it doesn’t include the text within the logo i.e. ‘SAP Kyma’ and hence on accessing logo , text within the logo i.e. ‘SAP Kyma’ is  not announced by the screen reader.
Screenshot:
 
Case 2: 
 

Expected Behavior:
An appropriate  text alternative must be provided for ‘SAP Kyma’ logo within header shell bar region  which includes the text within the logo i.e. ‘SAP Kyma’ and also on accessing logo , text within the logo i.e. ‘SAP Kyma’ must be announced by the screen reader along with other necessary information.

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-255.5 (Level AA), ACC-264.1 (Level A) _ Modify Modules _ Image of text _ Text alt not descriptive and text within image not read.docx](https://github.com/user-attachments/files/16659521/ACC-255.5.Level.AA.ACC-264.1.Level.A._.Modify.Modules._.Image.of.text._.Text.alt.not.descriptive.and.text.within.image.not.read.docx)
[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659523/Issue.processing-09.2021.002.docx)
